modules/core/04-channel/types/packet.go

```diff
@@ -26,6 +26,7 @@ func CommitPacket(packet Packet) []byte {
 	case IBC_VERSION_UNSPECIFIED, IBC_VERSION_1:
 		return commitV1Packet(packet)
```

Comment:
Can be of course done in a separate PR, but probably we can rename this to `commitPacketV1`?